guideline/cfh.md

```diff
@@ -131,26 +131,11 @@ off by the PAC before they can be put in place.
    Proposals will have to provide details about how these can be implemented at
    the proposed venues.
 
-   1. Food and drinks for lunches and breaks on all conference days (and
```

Comment:
I agree that a PyCon, at different scale and for different audience, do not need to provide lunch or beverage.  It's up to the team and we should not impose restriction.

But for PyCon APAC, I think lunch and beverage are important.  IMHO they are more important than good to have.  If one is asked to find food himself/herself in the lunch break of the conference, it's hard to make smooth intellectual engagement in the conference.

I agree PyCon itself is a social event, but it consists of intellectual components.  Again, the composition should be determined by the PyCon's team.  But for PyCon APAC, it's good to set up minimal.  I think lunches is part of the minimal requirement.  Dinners might not.